views/default/output/date.php

```diff
@@ -15,6 +15,6 @@
 
 try {
 	$dt = \Elgg\Values::normalizeTime($value);
-	echo $dt->format($format);
+	echo elgg_format_element('time', [], $dt->format($format));
```

Comment:
I vaguely remember adding a css rule for `time` which may make it look odd as a general output

Reply:
yes, it is styled like the friendly time... i think i can accept that for now... or do you want something else?

Reply:
The reason I wanted it wrapped is to denote normalized time in user's timezone. We don't have the support for that now, so maybe just leave it as it for now?

Reply:
i think wrapping it in `time` is semantically correct. we are not using the output view anywhere at this moment, so we could also improve/fix the styling later

Reply:
maybe add `'datetime' => date('c', $timestamp),` as an attribute on time

Reply:
`'title'` would be more appropriate as it will generate a tooltip. 
`c` is not a very human-friendly format. I would say use elgg_echo date + time format + `T`

Reply:
```php
'title' => date(elgg_echo('friendlytime:date_format'), $timestamp),
'datetime' => date('c', $timestamp),
```
is currently used on friendly time... could just go with that for consistency purpose

Reply:
We should just check what w3 spec says about `time` tag

Reply:
- https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time
- https://www.w3.org/TR/html51/infrastructure.html#dates-and-times

conclusion... just add `datetime = date('c')`